helmchart/otel-add-on/values.yaml

```diff
@@ -175,7 +167,8 @@ opentelemetry-collector:
             - resource.attributes["app_id"] == "nodeapp" # drop all metrics w/o the app_id = nodeapp tag on them
 
             # - not (IsMatch(name, "http/client/*")) # drop all metrics that don't start w/ http/client/
-            # more here: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/pkg/ottl/LANGUAGE.md & https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/pkg/ottl/ottlfuncs
+            # more here: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/pkg/ottl/LANGUAGE.md
```

Comment:
nit: maybe don't reference `main`? a release branch or tag might serve better in long term.